oscore-cf/src/main/java/org/eclipse/californium/oscore/OSCoreClient.java

```diff
@@ -0,0 +1,140 @@
+/*******************************************************************************
```

Comment:
will be obsolete using CoapStackFactory. So, please revert this change.

Reply:
Will be removed in re designed PR.